pmd-java/src/main/java/net/sourceforge/pmd/lang/java/symboltable/MethodNameDeclaration.java

```diff
@@ -32,6 +32,11 @@ public boolean isVarargs() {
         return p.isVarargs();
     }
 
+    public boolean isPrimitiveReturnType() {
+        return getMethodNameDeclaratorNode().getParent().getResultType().getChild(0)
```

Comment:
In case the method is `void`, there won't be any children and `getChild(0)` throws. We'll need to check with `isVoid()` for that case.

I'll update this when I merge.